discovery/gossiper_test.go

```diff
@@ -741,18 +741,18 @@ func createTestCtx(startHeight uint32) (*testCtx, func(), error) {
 			c := make(chan struct{})
 			return c
 		},
-		Router:                    router,
-		TrickleDelay:              trickleDelay,
-		RetransmitDelay:           retransmitDelay,
-		ProofMatureDelta:          proofMatureDelta,
-		WaitingProofStore:         waitingProofStore,
-		MessageStore:              newMockMessageStore(),
-		RotateTicker:              ticker.NewForce(DefaultSyncerRotationInterval),
-		HistoricalSyncTicker:      ticker.NewForce(DefaultHistoricalSyncInterval),
-		NumActiveSyncers:          3,
-		AnnSigner:                 &mockSigner{nodeKeyPriv1},
-		SubBatchDelay:             time.Second * 5,
-		MinimumBatchSize:          10,
+		Router:               router,
```

Comment:
Nit: this fixup should be applied to f4a9edca7e107e9af31ad8b44c8ed9c5f78c2792 since that's where the changes were originally introduced.